### ISSUE TITLE
`clickable: false` still responds to clicks on label

### ISSUE THREAD

**Issue by dgaspar** [NightCatSama/vue-slider-component#422]
In the example https://jsfiddle.net/6k3fo5dt/1/,
click handlers on `progress` are disabled, but not those on the `mark` labels.

Is that by design? Is there a way to disable them?
For my use case, the user should have to drag the dot.

**Comment by dgaspar** [comment]
As a workaround, I noticed that customizing the `label` slot made it stop listening to clicks:
```
<template v-slot:label="{ label, active }">
    <div :class="['vue-slider-mark-label', 'custom-label', { active }]">{{ label }}</div>
</template>
```

**Comment by NightCatSama** [comment]
Thanks for your feedback, fixed in version 3.0.45.

**Comment by dgaspar** [comment]
ok, I've updated it, removed the slot:label hack,
and now the labels are not responding to click anymore.
Thanks!
